src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        Random random = new Random(Color.values().length);
+        return Color.values()[random.nextInt()];
```

Comment:
The `random.nextInt()` method should be called with an argument specifying the upper bound. Use `random.nextInt(Color.values().length)` to get a valid index for the `Color.values()` array.